src/quotamodel.c

```diff
@@ -184,12 +188,27 @@ struct LocalRejectMapEntry
 };
 
 /* using hash table to support incremental update the table size entry.*/
-static HTAB *table_size_map = NULL;
+static HTAB       *table_size_map                  = NULL;
+static TimestampTz table_size_last_overflow_report = 0;
+
+#define TABLE_SIZE_MAP_WARNING                                             \
+	"[diskquota] the number of tables reached the limit, please increase " \
+	"the GUC value for diskquota.max_table_segments. Current "             \
+	"diskquota.max_table_segments value:"
 
 /* rejectmap for database objects which exceed their quota limit */
 static HTAB *disk_quota_reject_map       = NULL;
 static HTAB *local_disk_quota_reject_map = NULL;
 
+static TimestampTz disk_quota_reject_last_overflow_report       = 0;
+static TimestampTz local_disk_quota_reject_last_overflow_report = 0;
+
+#define DISK_QUOTA_REJECT_MAP_WARNING                               \
+	"[diskquota] Shared disk quota reject map size limit reached. " \
+	"Some out-of-limit schemas or roles will be lost in rejectmap. Current limit:"
```

Comment:
May be add info for increase some GUC?

Reply:
the disk_quota_reject_map limit is fixed MAX_DISK_QUOTA_REJECT_ENTRIES